ext/tunemygc/tunemygc_ext.c

```diff
@@ -39,26 +43,53 @@ static VALUE tunemygc_walltime(VALUE mod)
     return DBL2NUM(_tunemygc_walltime());
 }
 
+static void reverse(tunemygc_stat_record** head_ref) {
```

Comment:
Not sure what you mean, I must be missing something, I can only navigate one way...